### DISCUSSION TITLE
The dubfocus firmware appears every time I disconnect the board and reconnect...

### DISCUSSION THREAD

**Discussion by NommarsX** [shanteacontrols/OpenDeck#227] [General]
Hello, I need help, every time I turn off the board and connect it again my firmware is dubfocus, I update it. every day, choosing opendeck 2... through bootloader, but it seems that it does not work because every time I access the web configurator my board returns to be dubfocus, igor please can you tell me if there is any solution for this ? thanks.

**Comment by phillix2** [comment]
Hey! This problem was driving me nuts! It started to happen to me 2 days ago.

Instead of the Mega 2560, it now shows:

BoardDubFocus v1Firmwarevundefined.undefined.undefined

And I don't know if this is causing that as soon as I connect the arduino, it starts sending messages from the analog imputs I had enabled.

-- | --
07:44:06 255 | Component infoAnalog #0Raw data[ 240 0 83 67 1 0 73 3 0 0 247 ]  Dec
07:44:05 751 | Component infoAnalog #0Raw data[ 240 0 83 67 1 0 73 3 0 0 247 ]  Dec
07:44:05 242 | Component infoAnalog #0Raw data[ 240 0 83 67 1 0 73 3 0 0 247 ]  Dec
07:44:04 726 | Component infoAnalog #0Raw data[ 240 0 83 67 1 0 73 3 0 0 247 ]  Dec
07:44:04 209 | Component infoAnalog #0Raw data[ 240 0 83 67 1 0 73 3 0 0 247 ]  Dec
07:44:03 702 | Component infoAnalog #0Raw data[ 240 0 83 67 1 0 73 3 0 0 247 ]  Dec
07:44:03 194 | Component infoAnalog #0Raw data[ 240 0 83 67 1 0 73 3 0 0 247 ]  Dec
07:44:02 689 | Component infoAnalog #0Raw data[ 240 0 83 67 1 0 73 3 0 0 247 ]  Dec
07:44:02 181 | Component infoAnalog #0Raw data[ 240 0 83 67 1 0 73 3 0 0 247 ]  Dec
07:44:01 677 | Component infoAnalog #0Raw data[ 240 0 83 67 1 0 73 3 0 0 247 ]  Dec
07:44:01 164 | Component infoAnalog #1Raw data[ 240 0 83 67 1 0 73 3 0 1 247 ]  Dec
07:44:00 664 | Component infoAnalog #0Raw data[ 240 0 83 67 1 0 73 3 0 0 247 ]  Dec
07:44:00 154 | Component infoAnalog #0Raw data[ 240 0 83 67 1 0 73 3 0 0 247 ]  Dec
07:43:59 636 | Component infoAnalog #0Raw data[ 240 0 83 67 1 0 73 3 0 0 247 ]  Dec
07:43:59 114 | Component infoAnalog #0Raw data[ 240 0 83 67 1 0 73 3 0 0 247 ]  Dec
07:43:58 593 | Component infoAnalog #1Raw data[ 240 0 83 67 1 0 73 3 0 1 247 ]  Dec
07:43:58 86 | Component infoAnalog #1Raw data[ 240 0 83 67 1 0 73 3 0 1 247 ]  Dec
07:43:57 580 | Component infoAnalog #0Raw data[ 240 0 83 67 1 0 73 3 0 0 247 ]  Dec
07:43:57 56 | Component infoAnalog #1Raw data[ 240 0 83 67 1 0 73 3 0 1 247 ]  Dec
07:43:33 574 | Error Code 720 Received a request not matching the active one.
07:43:33 565 | Error Code 720 Received a request not matching the active one.
07:43:33 554 | Error Code 720 Received a request not matching the active one.
07:43:33 547 | Request 118Error Code 9 This error is returned when wanted parameter is incorrect.
07:43:33 536 | Error Code 720 Received a request not matching the active one.
07:43:33 527 | Error Code 720 Received a request not matching the active one.
07:43:33 520 | Error Code 720 Received a request not matching the active one.
07:43:33 507 | Error Code 720 Received a request not matching the active one.
07:43:33 496 | Error Code 720 Received a request not matching the active one.
07:43:33 487 | Error Code 720 Received a request not matching the active one.
07:43:33 481 | Error Code 720 Received a request not matching the active one.
07:43:33 480 | Error Code 720 Received a request not matching the active one.
07:43:33 456 | Error Code 720 Received a request not matching the active one.
07:43:33 447 | Error Code 720 Received a request not matching the active one.
07:43:33 439 | Error Code 720 Received a request not matching the active one.
07:43:33 427 | Error Code 720 Received a request not matching the active one.
07:43:33 416 | Error Code 720 Received a request not matching the active one.
07:43:33 394 | Error Code 716 Failed to decode request data.
07:43:33 393 | Error Code 720 Received a request not matching the active one.

**Reply by phillix2** [reply]
Update: I connected the pins to the potentiometer and it made the board stop sending those constant messages; then I could apply the "reset to factory" option in the Control UI, chrome crashed so I stopped it from the task manager, and disconnect-connect the arduino.

After opening again the control UI, the mega was correctly recognized.

**Comment by paradajz** [comment]
Which board revision do you have? It's listed on the back side.

**Reply by phillix2** [reply]
The backside says >Mega2560 V3E, the box of the arduino says R3

**Reply by phillix2** [reply]
I started to see the same label again in the web configurator:
BoardDubFocus v1Firmwarevundefined.undefined.undefined

In the device manager, the OpenDeck| mega2560 is listed under sound, video and game controllers, it is recognized as a Midi controller by the software synths, and the already programmed controls work as expected.

**Reply by paradajz** [reply]
I've actually asked @NommarsX

**Comment by NommarsX** [comment]
Rev 2.2.1
Thanks

**Comment by paradajz** [comment]
So RP2040 based then. Unplug the board from your PC, and short circuit the RPB 2-pin header located below the USB C port. Connect the board to the PC and a new disk drive will show called RP-RP2. Make sure that the short circuit is removed once the drive is shown. Drag the opendeck22.uf2 file to the drive which you can download from [here](https://github.com/shanteacontrols/OpenDeck/releases/tag/v7.5.3). After a few seconds the board will reboot. Let me know if it works.

**Comment by NommarsX** [comment]
Hi, I did as indicated in the message, when I open the web editor my badge
is still dubfocus... thanks.

**Reply by paradajz** [reply]
That's pretty weird. Did you make that board by yourself or did you bought it from me?

**Comment by NommarsX** [comment]
I bought it from you. if it's strange, when I load the .rp2 file when
connecting to the pc for the first time in the editor it looks like
opendeck board... but if I disconnect from the pc and reconnect it's
already dubfocus... it's weird. I have tried to restore from the factory
and it remains the same... I appreciate any help, I have also noticed that
I configure any control, for example in the analog section "pin 0" when I
finish configuring I go to the next "pin 1" when I return to 0 I have I
have to re-enable it and it's as if I hadn't done any configuration... I've
tried all the possible browsers in case it was a Chrome problem... but it
works the same with all of them. All the best

**Comment by paradajz** [comment]
Please contact me via email.
